contracts/staking/StakingInfo.sol

```diff
@@ -41,10 +41,12 @@ contract IStakeManager {
 
 contract StakingInfo {
     using SafeMath for uint256;
+    mapping(uint256 => uint256) public validatorNonce;
```

Comment:
Shouldn't we put this in `StakeManagerStorage`?  Other than everything looks good. 

Also `validatorNonce` in `StakingInfo.sol` is okay 😄

Reply:
`StakeManagerStorage` is used in many other places where the nonce will not make sense. @0xAshish has more context.